wordpress_performance_best_practices.md

```diff
@@ -30,12 +30,13 @@ If you start seeing a surge of user traffic hitting your site, you should consid
 * WordPress on Windows
 	* [IIS output caching](https://www.iis.net/learn/manage/managing-performance-settings/walkthrough-iis-output-caching): It’s easy to configure and setup. It significantly improves site throughput but may prevent confirmation messages from showing up when submitting comments. To learn more about IIS output caching, click here.
 	* [WP Super cache](https://wordpress.org/plugins/wp-super-cache/): It significantly improves site throughput and correctly handles comments submissions and other visitors’ actions. It slightly more complex to setup and configure than compared to IIS output caching. To learn more about Super cache configuration on IIS click here.
-	* [App Service Dynamic Cache](https:/github.com/projectkudu/kudu/wiki/Configurable-settings#turning-on-the-dynamic-cache-feature): turn on the Dynamic cache to improve performance. 
+	* [App Service Dynamic Cache](https://learn.microsoft.com/en-us/azure/app-service/reference-app-settings?tabs=kudu%2Cdotnet#caching): turn on the Dynamic Cache to improve file system performance. 
```

Comment:
The URL should be:

`https://learn.microsoft.com/azure/app-service/reference-app-settings?tabs=kudu%2Cdotnet#caching`

and not:

`https://learn.microsoft.com/**en-us/**azure/app-service/reference-app-settings?tabs=kudu%2Cdotnet#caching`

This allows localization re-direction to resolve to the right language for the customer if a localized copy of the doc is available.

Reply:
Ah, that makes sense, thank you for the catch. I updated it.